package-lock.json

```diff
@@ -31,25 +31,24 @@
       "name": "azure-rest-api-specs-eng-tools",
       "dev": true,
       "devDependencies": {
+        "@azure-tools/specs-model": "file:specs-model",
         "@azure-tools/suppressions": "file:suppressions",
         "@azure-tools/typespec-requirement": "file:typespec-requirement",
         "@azure-tools/typespec-validation": "file:typespec-validation"
       }
     },
     "eng/tools/node_modules/@types/node": {
       "version": "18.19.39",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-18.19.39.tgz",
-      "integrity": "sha512-nPwTRDKUctxw3di5b4TfT3I0sWDiWoPQCZjXhvdkINntwr8lcoVCKsTgnXeRubKIlfnV+eN/HYk6Jb40tbcEAQ==",
       "dev": true,
+      "license": "MIT",
```

Comment:
Looks like a bad update to package-lock.json?

Reply:
Unsure what happened here. I am also running into this now:

https://github.com/Azure/azure-rest-api-specs/actions/runs/9947828905/job/27481266070?pr=29820#step:4:12

```
Error: npm error tsconfig.json(5,5): error TS6053: File '@tsconfig/strictest/tsconfig.json' not found.
npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2024-07-15T22_59_16_546Z-debug-0.log
Error: Process completed with exit code 1.
```

It did work before, so I am yet unsure what's up.

Reply:
> Unsure what happened here. I am also running into this now:
> 
> https://github.com/Azure/azure-rest-api-specs/actions/runs/9947828905/job/27481266070?pr=29820#step:4:12
> 
> ```
> Error: npm error tsconfig.json(5,5): error TS6053: File '@tsconfig/strictest/tsconfig.json' not found.
> npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2024-07-15T22_59_16_546Z-debug-0.log
> Error: Process completed with exit code 1.
> ```
> 
> It did work before, so I am yet unsure what's up.

I pushed changes to fix `package-lock.json` in this PR.  How I did it:

1. `git clean -xdf`
  - remove all cached files in your local repo folder
2.  `cp ../specs-main/package-lock.json ./`
  - Overwrite `package-lock.json` in local folder with copy from main
3. `npm install`
  - Should make minimal changes to `package-lock.json`, to support whatever dep changes you make in `package.json`.